src/common/client.go

```diff
@@ -318,3 +319,21 @@ func handleRepositories(client *opslevel.Client, registration ServiceRegistratio
 		}
 	}
 }
+
+func handleSystem(client *opslevel.Client, registration ServiceRegistration, service *opslevel.Service) {
+	if registration.System != "" {
+		system, foundSystemErr := client.GetSystem(registration.System)
+		if foundSystemErr != nil {
+			log.Warn().Msgf("[%s] System with alias: '%s' not found so service cannot be attached to it ... skipping", service.Name, system)
+			return
+		}
+
+		assignServiceErr := system.AssignService(client, service.Aliases[0])
```

Comment:
I thnk using the Id to uniquely identify might be clearer.g. https://github.com/OpsLevel/kubectl-opslevel/blob/692346d87380b028c8197834e2bc49ad36faef88/src/common/client.go#L239

```suggestion
		assignServiceErr := system.AssignService(client, string(service.Id))
```